course1/semester2/hw3/task1/src/c1_s2/konovalov/task3_1/sorterTest.java

```diff
@@ -0,0 +1,66 @@
+package c1_s2.konovalov.task3_1;
+
+import org.junit.Test;
+import org.junit.Before;
+import org.junit.After;
+import org.junit.Assert;
+import java.util.Random;
+
+public class SorterTest {
```

Comment:
Error:(9, 8) java: class SorterTest is public, should be declared in a file named SorterTest.java